client.bal

```diff
@@ -0,0 +1,296 @@
+        string timestamp = check self.generateTimestamp();
+        string signatureString = check self.generateSignature(buildPayload(parameters), timestamp, parameters["Action"].toString());
+        http:Request|error request = self.generateRequest(buildPayload(parameters), timestamp, signatureString, parameters["Action"].toString());
+        string queryParameters = check self.generateQueryParameters(buildPayload(parameters), timestamp, signatureString);
+        xml response = check sendRequest(self.amazonSimpleDBClient, request, queryParameters);
+        SelectResponse selectResponse = check xmlToSelectResponse(response);
+        return selectResponse;
+    }
+
+    # list available domains.
+    #
+    # + return - `ListDomainsResponse` on success else an `error`
+    remote isolated function listDomains() returns @tainted ListDomainsResponse|error {
+        map<string> parameters = {};     
+        parameters["Action"] = "ListDomains";
+        string timestamp = check self.generateTimestamp();
+        string signatureString = check self.generateSignature(buildPayload(parameters), timestamp, parameters["Action"].toString());
+        http:Request|error request = self.generateRequest(buildPayload(parameters), timestamp, signatureString, parameters["Action"].toString());
+        string queryParameters = check self.generateQueryParameters(buildPayload(parameters), timestamp, signatureString);
+        xml response = check sendRequest(self.amazonSimpleDBClient, request, queryParameters);
+        io:println(response);
+        ListDomainsResponse listDomainsResponse = check xmlToListsDomain(response);
+        return listDomainsResponse;
+    }
+
+    # Delete a domain.
+    #
+    # + domainName - Name of domain
+    # + return - `DeleteDomainResponse` on success else an `error`
+    remote isolated function deleteDomain(string domainName) returns @tainted DeleteDomainResponse|error {
+        map<string> parameters = {};
+        parameters["Action"] = "DeleteDomain";
+        parameters["DomainName"] = domainName;
+        string timestamp = check self.generateTimestamp();
+        string signatureString = check self.generateSignature(buildPayload(parameters), timestamp, parameters["Action"].toString());
+        http:Request|error request = self.generateRequest(buildPayload(parameters), timestamp, signatureString, parameters["Action"].toString());
+        string queryParameters = check self.generateQueryParameters(buildPayload(parameters), timestamp, signatureString);
+        xml response = check sendRequest(self.amazonSimpleDBClient, request, queryParameters);
+        DeleteDomainResponse deletedDomainResponse = check xmlToDeletedDomain(response);
+        return deletedDomainResponse;
+    }
+
+    # Get all of the attributes associated with the item.
+    #
+    # + domainName - Name of domain
+    # + itemName - Name of item
+    # + consistentRead - Denote whether consistent reads are acceptable or not
```

Comment:
True if consistent reads are to be accepted

Reply:
Fixed